src/Resources/contao/dca/tl_content.php

```diff
@@ -716,13 +716,25 @@
 		(
 			'label'                   => &$GLOBALS['TL_LANG']['tl_content']['articleAlias'],
 			'exclude'                 => true,
-			'inputType'               => 'select',
-			'options_callback'        => array('tl_content', 'getArticleAlias'),
-			'eval'                    => array('mandatory'=>true, 'chosen'=>true, 'submitOnChange'=>true, 'tl_class'=>'w50 wizard'),
-			'wizard' => array
-			(
-				array('tl_content', 'editArticleAlias')
```

Comment:
You should also remove this method from the class.

Reply:
I have not removed it for BC reasons, some extensions could use it. Maybe instead of removing it I should throw the deprecation warning?

Reply:
Agreed, we should keep it. But we should add the deprecation notice.

Reply:
Changed in 0d178e6.